lib/views/pages/newsfeed/addPost.dart

```diff
@@ -1,210 +0,0 @@
-//flutter imported packages
```

Comment:
Instead of deleting entire file, can you add only changes required to fix the linter issue ?